pgml-extension/src/bindings/transformers/whitelist.rs

```diff
@@ -145,18 +152,18 @@ mod tests {
     #[pg_test]
     fn test_trusted_model() {
         let model = "Salesforce/xgen-7b-8k-inst";
-        set_config(CONFIG_HF_WHITELIST, model).unwrap();
-        set_config(CONFIG_HF_TRUST_WHITELIST, model).unwrap();
+        set_config(PGML_HF_WHITELIST.0, model).unwrap();
+        set_config(PGML_HF_TRUST_WHITELIST.0, model).unwrap();
 
         let task_json = format!(json_template!(), model, false);
         let task: Value = serde_json::from_str(&task_json).unwrap();
         assert!(verify_task(&task).is_ok());
 
         let task_json = format!(json_template!(), model, true);
         let task: Value = serde_json::from_str(&task_json).unwrap();
-        assert!(verify_task(&task).is_ok());
+        assert!(verify_task(&task).is_err());
```

Comment:
In the initial commit of this commit, this step should fails with `RemoteCodeNotTrusted`.

https://github.com/postgresml/postgresml/blob/f4e87c5b85013ec74ec583e33343745396320442/pgml-extension/src/bindings/transformers/whitelist.rs#L189